projects/Make/README.md

```diff
@@ -0,0 +1,31 @@
+# raylib-cpp Make Example Project
+
+The [Raylib C++ Starter Kit](https://github.com/CapsCollective/raylib-cpp-starter), put together by [CapsCollective](https://caps-collective.itch.io), uses [`make`](https://www.gnu.org/software/make/) to compile a raylib-cpp starter project.
+
+## Build
+
+1. Check out the repository
+    ```
+    git clone https://github.com/CapsCollective/raylib-cpp-starter.git
+    ```
+
+2. Build the project
+    ```
+    cd raylib-cpp-starter
+
+    # Linux
```

Comment:
I would just change line to say `# Linux & macOS` to make it clear.

Reply:
``` suggestion
    # Linux & macOS
```